cmd/podman/common/specgen.go

```diff
@@ -529,6 +529,10 @@ func FillOutSpecGen(s *specgen.SpecGenerator, c *ContainerCLIOpts, args []string
 			case "seccomp":
 				s.SeccompProfilePath = con[1]
 				s.Annotations[define.InspectAnnotationSeccomp] = con[1]
+			case "mask":
+				s.ContainerSecurityConfig.Mask = con[1]
+			case "unmask":
+				s.ContainerSecurityConfig.Unmask = con[1]
 			default:
```

Comment:
this case is getting long enough that it might be nice to alphabetize it all.

Reply:
Fixed